gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -633,15 +625,12 @@ def _get_variant_calls(self, variant_proto):
       else:
         call.genotype = list(call_proto.genotype)
 
-      phaseset_from_format = (
-          self._convert_list_value(call_proto.info[PHASESET_FORMAT_KEY], False)
-          if call_proto.info.get(PHASESET_FORMAT_KEY, None)
-          else None)
-      # Note: Call is considered phased if it contains the 'PS' key regardless
-      # of whether it uses '|'.
-      if phaseset_from_format or call_proto.is_phased:
-        call.phaseset = (phaseset_from_format if phaseset_from_format
-                         else DEFAULT_PHASESET_VALUE)
+      if call_proto.is_phased:
```

Comment:
I modified this if statement, I think this is more straightforward. Let me know what you think.